front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -155,6 +155,19 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
     }
   }, [isCreateStudyError, isPatchStudyError, isDeleteStudyError]);
 
+  const isInvalidDate = () => {
+    if (currentStudy.start_date) {
+      const startDate = new Date(currentStudy.start_date);
+      if (currentStudy.expected_end_date && new Date(currentStudy.expected_end_date) < startDate) {
+        return true;
+      }
+      if (currentStudy.actual_end_date && new Date(currentStudy.actual_end_date) < startDate) {
+        return true;
+      }
+    }
+    return false;
+  };
```

Comment:
You can directly compare the string values :) 

```suggestion
 const isInvalidDate = () => {
    if (currentStudy.start_date) {
      const startDate = currentStudy.start_date;
      if (currentStudy.expected_end_date && currentStudy.expected_end_date < startDate) return true;
      if (currentStudy.actual_end_date && currentStudy.actual_end_date < startDate) return true;
    }
    return false;
  };
```

Reply:
Ty, fixed